### DISCUSSION TITLE
Can I use driver/rmt.h in natmod? Or do I have to use external C modules?

### DISCUSSION THREAD

**Discussion by MarkHmnv** [micropython/micropython#15480] [Libraries & Drivers]
Can I use [driver/rmt.h](https://docs.espressif.com/projects/esp-idf/en/v4.4.8/esp32/api-reference/peripherals/rmt.html) in [natmod](https://docs.micropython.org/en/latest/develop/natmod.html)? Or do I have to use [external C modules](https://docs.micropython.org/en/latest/develop/cmodules.html)?

**Comment by sophiedegran** [comment]
Have a look to awesome-micropython.org/#ir
